sdks/python/src/opik/cli/export.py

```diff
@@ -2,30 +2,462 @@
 
 import csv
 import json
-import re
 import sys
 from datetime import datetime
 from pathlib import Path
-from typing import Dict, List, Optional
+from typing import Any, Callable, Dict, List, Optional
 
 import click
 from rich.console import Console
 from rich.progress import Progress, SpinnerColumn, TextColumn
 
 import opik
+from opik.api_objects.dataset.dataset import Dataset
```

Comment:
Please follow the rules described in sdks/python (it looks like they are not working unless you reference them now).
Here in particular I refer to importing modules instead of names.

Reply:
I don't know why the rules were not followed. In this case, I don't think using `dataset.Dataset` is better. There is only one Dataset here.